### PR TITLE
Update line height for text fields according to 1.7 version of guides

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/894
[Problem] Wrong line height
[Solution] Apply line height according to the guideline

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

File: src/css/profile/mobile/common/textinput.less [hunk 1]
```diff
@@ -103,7 +103,7 @@ textarea.ui-text-input {
 	box-sizing: border-box;
 	display: block;
 	width: 100%;
-	line-height: 54 * @unit_base;
+	line-height: 26 * @sp_base;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/textinput.less]: Do we use the @sp_base for the 'line-height' not @px_base?

Comment (lslachciak) [reply src/css/profile/mobile/common/textinput.less:106]: yes, in some places of guide I see font size values marked with sp so I assume all guide values refer to sp

Comment (lslachciak) [reply src/css/profile/mobile/common/textinput.less]: @HunseopJeong you are right. For Line height we should use px

**Comment by lslachciak** [comment]
updated PR according to @HunseopJeong comment

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Review by TomaszLukawskiSam — APPROVED** [review]

**Pull request merged**